### PR TITLE
Record standard deviations for multi-tag pose

### PR DESCRIPTION
**Pull request by mcm001**

### CODE REVIEW

File: photon-client/src/components/dashboard/tabs/TargetsTab.vue [hunk 1]
```diff
@@ -80,14 +110,16 @@ const currentPipelineSettings = useCameraSettingsStore().currentPipelineSettings
         </template>
       </v-simple-table>
     </v-row>
-    <v-row
+    <template
```
Comment (srimanachanta) [inline photon-client/src/components/dashboard/tabs/TargetsTab.vue:113]: This shouldn't be a template, if you don't want the `v-row` styling use a div

Comment (mcm001) [reply photon-client/src/components/dashboard/tabs/TargetsTab.vue:113]: I want the if to apply to both, tho

File: photon-client/src/components/dashboard/tabs/TargetsTab.vue [hunk 2]
```diff
@@ -2,8 +2,38 @@
...
 
-const currentPipelineSettings = useCameraSettingsStore().currentPipelineSettings;
+let oldResults: {targets: Transform3d[]} = Vue.observable({ targets: [] });
+let stdev = ref({
+  x: 0,
+  y: 0,
+  z_angle: 0
+});
+
+const standardDeviation = (arr, usePopulation = false) => {
```
Comment (srimanachanta) [inline photon-client/src/components/dashboard/tabs/TargetsTab.vue:15]: type arr as `number[]`

nit: get rid of the `usePopulation `prop as it is always false

File: photon-client/src/components/dashboard/tabs/TargetsTab.vue [hunk 3]
```diff
@@ -2,8 +2,38 @@
 import { useCameraSettingsStore } from "@/stores/settings/CameraSettingsStore";
 import { PipelineType } from "@/types/PipelineTypes";
 import { useStateStore } from "@/stores/StateStore";
+import Vue, { ref, watch } from "vue";
+import type { Transform3d } from "@/types/PhotonTrackingTypes";
 
-const currentPipelineSettings = useCameraSettingsStore().currentPipelineSettings;
+let oldResults: {targets: Transform3d[]} = Vue.observable({ targets: [] });
```
Comment (srimanachanta) [inline photon-client/src/components/dashboard/tabs/TargetsTab.vue:8]: Don't think this needs to be like this, I'm pretty sure you can just do `const targets = []`

File: photon-client/src/components/dashboard/tabs/TargetsTab.vue [hunk 4]
```diff
@@ -2,8 +2,38 @@
...
+let stdev = ref({
```
Comment (srimanachanta) [inline photon-client/src/components/dashboard/tabs/TargetsTab.vue:9]: should be a const

**Review by srimanachanta — APPROVED** [review]

**Pull request merged**